packages/backend/.default.env

```diff
@@ -2,6 +2,21 @@
 ### These configs are related to the configuration of the interaction with the
 ### AWS S3 bucket used as storage for ceremony artifacts.
 
+PROJECT_ID="YOUR-FIREBASE-PROJECT-ID"
```

Comment:
You have probably copied outdated code here. The `.default.env` should look like this https://github.com/quadratic-funding/mpc-phase2-suite/blob/dev/packages/backend/.default.env